script/QA/QA_01.txt

```diff
@@ -26,7 +26,7 @@ Welcome to 'Teach Me, Ciel-Sensei!'.%{n}
 [sha:cc86c2e84e7ec326d805dedf7a0261b93795a15b]{
 -- Page 4, Offset 42771.
 -- 教官のシエル（本編後、18歳想定）です。
-Instructor Ciel here.%{n}(Presumed to be 18 years old)
+Instructor Ciel here.%{n}(Presumed to be eighteen years old)
```

Comment:
You checked this fits?

Reply:
I will now

Reply:
looks good to me
![image](https://user-images.githubusercontent.com/90473809/235365243-31373397-3cd0-4279-8ec0-33e8dc2e3d8c.png)